docs/translations/translation-management.md

```diff
@@ -0,0 +1,68 @@
+# Translation management
+
+As mentioned in [Translation basics](docs/translations/translation-basics.md), all translations are managed using [GlotPress](https://wordpress.org/plugins/glotpress/). The translations of the WooCommerce Blocks plugin can be found on https://translate.wordpress.org/projects/wp-plugins/woo-gutenberg-products-block/.
+
+## Roles
+
+### Translation Contributors
+
+A Translation Contributors can suggest translations. These suggested translations need to be verified by a GTE or a PTE.
```

Comment:
Maybe write the full definition of GTE and PTE to begin with in case people are not familiar with the acronym. I know you define it below but I think it should be written in full first, what do you think?

Reply:
I've added a TOC at the top of the page, that allows the reader to jump directly to the corresponding section. I also changed `GTE` to `General Translation Editor (GTE)` and `PTE` to `Project Translation Editor (PTE)` in the first section.